EIPS/eip-7002.md

```diff
@@ -38,92 +39,92 @@ Note, 0x00 withdrawal credentials can be changed into 0x01 withdrawal credential
 
 | Name | Value | Comment |
 | - | - | - |
-| `VALIDATOR_EXIT_ADDRESS` | `0x0f1ee3e66777F27a7703400644C6fCE41527E017` | Where to call and store relevant details about exit mechanism |
+| `WITHDRAWAL_REQUEST_PREDEPLOY_ADDRESS` | `0xEd8EA01d70Cb49726175BCf2778B9C982912e017` | Where to call and store relevant details about exit / partial withdrawal mechanism |
 | `SYSTEM_ADDRESS` | `0xfffffffffffffffffffffffffffffffffffffffe` | Address used to invoke system operation on contract
-| `EXCESS_EXITS_STORAGE_SLOT` | 0 | |
-| `EXIT_COUNT_STORAGE_SLOT` | 1 | |
-| `EXIT_MESSAGE_QUEUE_HEAD_STORAGE_SLOT` | 2 | Pointer to head of the exit message queue |
-| `EXIT_MESSAGE_QUEUE_TAIL_STORAGE_SLOT` | 3 | Pointer to the tail of the exit message queue|
-| `EXIT_MESSAGE_QUEUE_STORAGE_OFFSET` | 4 | The start memory slot of the in-state exit message queue|
-| `MAX_EXITS_PER_BLOCK` | 16 | Maximum number of exits that can be dequeued into a block |
-| `TARGET_EXITS_PER_BLOCK` | 2 | |
-| `MIN_EXIT_FEE` | 1 | |
-| `EXIT_FEE_UPDATE_FRACTION` | 17 | |
+| `EXCESS_WITHDRAWAL_REQUESTS_STORAGE_SLOT` | 0 | |
+| `WITHDRAWAL_REQUEST_COUNT_STORAGE_SLOT` | 1 | |
+| `WITHDRAWAL_REQUEST_QUEUE_HEAD_STORAGE_SLOT` | 2 | Pointer to head of the withdrawal request message queue |
+| `WITHDRAWAL_REQUEST_QUEUE_TAIL_STORAGE_SLOT` | 3 | Pointer to the tail of the withdrawal request message queue|
+| `WITHDRAWAL_REQUEST_QUEUE_STORAGE_OFFSET` | 4 | The start memory slot of the in-state withdrawal request message queue|
+| `MAX_WITHDRAWAL_REQUESTS_PER_BLOCK` | 16 | Maximum number of withdrawal requests that can be dequeued into a block |
+| `TARGET_WITHDRAWAL_REQUESTS_PER_BLOCK` | 2 | |
+| `MIN_WITHDRAWAL_REQUEST_FEE` | 1 | |
+| `WITHDRAWAL_REQUEST_FEE_UPDATE_FRACTION` | 17 | |
 
 ### Execution layer
 
 #### Definitions
 
 * **`FORK_BLOCK`** -- the first block in a blockchain with the `timestamp` greater or equal to `FORK_TIMESTAMP`.
 
-#### Exit operation
+#### Withdrawal request operation
 
-The new exit operation consists of the following fields:
+The new withdrawal request operation is an [EIP-7685](./eip-7685.md) request
+with type `0x01` and consists of the following fields:
 
-1. `source_address: Bytes20`
-2. `validator_pubkey: Bytes48`
+1. `source_address`: `Bytes20`
+2. `validator_pubkey`: `Bytes48`
+3. `amount:` `uint64`
 
-RLP encoding of an exit **MUST** be computed as the following:
+The [EIP-7685](./eip-7685) encoding of a withdrawal request **MUST** be computed as the follows:
 
 ```python
-rlp_encoded_exit = RLP([
-    source_address,
-    validator_pubkey,
-])
+encoded = 0x01 ++ rlp([source_address, validator_pubkey, amount])
```

Comment:
A slight preference to have `WITHDRAWAL_REQUEST_TYPE` to get rid of the magic number

Reply:
sgtm